segmentstore/server/src/main/java/io/pravega/segmentstore/server/containers/StreamSegmentContainer.java

```diff
@@ -723,6 +723,14 @@ public boolean isOffline() {
         return flusher.flushToStorage(timeout);
     }
 
+    @Override
+    public CompletableFuture<Void> flushToStorage(int containerId, Duration timeout) {
```

Comment:
I think I missed this originally. This API doesn't make much sense. Why would we pass a `containerId` to this method? The `StreamSegmentContainer` already has its own id. How can it handle a different `containerId`?

Please remove this parameter from the API

Reply:
I've removed this parameter.